lib/exercises.js

```diff
@@ -20,10 +27,56 @@ function top_k_frequent_elements(list, k) {
 //   Each element can either be a ".", or a digit 1-9
 //   The same digit cannot appear twice or more in the same 
 //   row, column or 3x3 subgrid
-// Time Complexity: ?
-// Space Complexity: ?
+// Time Complexity: O(n), where n is the number of elements in the sudoku table
```

Comment:
I would say instead O(n<sup>2</sup>) where n is the length of a side of the sudoku table.